releases/7.5.3.md

```diff
@@ -14,6 +14,7 @@ Arches 7.5.3 Release Notes
 - Fixes print from resource manager #11035
 - Displays configured maximum file size and accepted image file formats in photo gallery #11022
 - Alphabetically sort graphs by name in search Resource Type filter #10646
+- Fixes bug stopping Url datatype widget input boxes updating when adding and editing in multi-card workflow step #11136
```

Comment:
It's good to avoid adding new lines to the end of the list just to avoid merge conflicts.